### ISSUE TITLE
More blurry results on higher resolution images

### ISSUE THREAD

**Issue by LaFeuilleMorte** [city-super/Octree-GS#55]
8K resolution:

![image](https://github.com/user-attachments/assets/88ea8846-78ae-49f8-b2f0-b8058e4ecc72)

2K resolution:

![image](https://github.com/user-attachments/assets/e2b4d647-2acd-4978-91d4-7df331f12e2e)

Both use the same hyper parameters and the same initial point cloud and camera pose params

**Comment by LaFeuilleMorte** [comment]
> 8K resolution:
> 
> ![image](https://private-user-images.githubusercontent.com/43473814/374785225-88ea8846-78ae-49f8-b2f0-b8058e4ecc72.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0NDI5NzcsIm5iZiI6MTcyODQ0MjY3NywicGF0aCI6Ii80MzQ3MzgxNC8zNzQ3ODUyMjUtODhlYTg4NDYtNzhhZS00OWY4LWIyZjAtYjgwNThlNGVjYzcyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA5VDAyNTc1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2NDk4MGZiMjYxY2U0MWY3NTU4Yzc0YzM5MGJmY2IwNTk2YWMwODNlOWU5NDczZGMxN2UzZTQ4NTkwNmQ0NTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.MO3attG1K9FDRNm31AzZNlpoaY_H-frooSBRf6Iii4Y)
> 
> 2K resolution:
> 
> ![image](https://private-user-images.githubusercontent.com/43473814/374785531-e2b4d647-2acd-4978-91d4-7df331f12e2e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0NDI5NzcsIm5iZiI6MTcyODQ0MjY3NywicGF0aCI6Ii80MzQ3MzgxNC8zNzQ3ODU1MzEtZTJiNGQ2NDctMmFjZC00OTc4LTkxZDQtN2RmMzMxZjEyZTJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA5VDAyNTc1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBiMjYwMTViZjYyZThhMGQ2ODYzMGExNjQwNDA5M2MyYjA2YjYyYWViNDY0YWU4ZWZkNDE0NzJmNGJiMzk5ZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t6Naeb0WedHNidD2zESiNKsopaGdRMylle0pZ9IouTI)
> 
> Both use the same hyper parameters and the same initial point cloud and camera pose params

I had an idea, If I turn on the warm up and progressive option, would it be better? As in my experiment, when training with much higher resolution images, the loss will be very hard to drop. So I guess warm up in lower resolution will help the gaussians to fit the scene.

**Comment by jianglh-WHU** [comment]
Undoubtedly, warming up in lower resolutions will significantly enhance visual performance. Several studies have provided evidence supporting this claim, i.e. [MVGS](https://arxiv.org/pdf/2410.02103v1),  [Spec-Gaussian](https://arxiv.org/pdf/2402.15870), [SuperGS](https://arxiv.org/abs/2410.02571).

However, I have a question: why do you insist on training with 8k images? When comparing 8k and 2k images on a standard monitor, the difference is often imperceptible.

Best regards,
lihan

**Comment by LaFeuilleMorte** [comment]
> Undoubtedly, warming up in lower resolutions will significantly enhance visual performance. Several studies have provided evidence supporting this claim, i.e. [MVGS](https://arxiv.org/pdf/2410.02103v1), [Spec-Gaussian](https://arxiv.org/pdf/2402.15870), [SuperGS](https://arxiv.org/abs/2410.02571).
> 
> However, I have a question: why do you insist on training with 8k images? When comparing 8k and 2k images on a standard monitor, the difference is often imperceptible.
> 
> Best regards, lihan

My dataset was 8K resolution. And I just expected finer details when zooming in lower level. Coz my dataset are images captured by a drone. So the scenes are very far away from the cameras. When I use 2K resolution, the pixels near the ground  becomes blurry when I zoom in the picture.  Did you mean even if I use 8k images, there won't be any gains in the rendering quality?

**Comment by jianglh-WHU** [comment]
If you want to obtain fine-grained details when zooming in, higher resolution is intuitively better. However, it will lead to two main problems:
1. GPU Memory: The higher the resolution you use, the more Gaussian primitives will be produced, resulting in greater GPU memory usage.
2. Training Time: Training at 8K resolution will take much longer than at 2K resolution.

**Comment by LaFeuilleMorte** [comment]
> Spec-Gaussian

Yeah, that's correct. training on 8K are always around 10 times slower than in 2K cases.  As for memory, I will load the dataset in RAM to save memory.

**Comment by HelloBigLuo** [comment]
hello，can u share how to load the dataset in RAM to save memory?
